### DISCUSSION TITLE
UR bar

### DISCUSSION THREAD

**Discussion by SteveBoel** [ppy/osu#13936] [Ideas] [answered]
Currently ver.2021.707.0, there's no way to make the UR bar bigger without making scaling the UI as well. I think having the option to just scale the UR bar size up and down is a good idea

**Comment by smoogipoo** [comment, marked as answer]
Open the skin editor and drag the handles.

**Reply by SteveBoel** [reply]
How do I open the skin editor? I don't see it in the options menu

**Reply by peppy** [reply]
Ctrl+Shift+S is the default binding. you can change it from the key bindings section.

**Reply by SteveBoel** [reply]
thx, very cool peppy
